wsi_deid/web_client/views/ItemView.js

```diff
@@ -336,6 +377,7 @@ wrap(ItemView, 'render', function (render) {
             this.events['change .g-hui-redact'] = flagRedaction;
             this.events['click a.g-hui-redact'] = flagRedaction;
             this.events['click .g-hui-redact-square-span'] = flagRedaction;
+            this.events['focusout .wsi-deid-replace-value'] = flagRedaction;
```

Comment:
Elsewhere I've used the `change` event rather than `focusout`.  Is there a virtue to one over the other in this case?

Reply:
I was afraid that `change` would trigger for every keystroke in the input field. I did some testing locally, and it appears as though this is not the case. I've switched from `focusout` to `change` for consistency with existing events.

Reply:
I think the `input` event is triggered for each keystroke and `change` only on loss of focus AND a difference in value.